CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## {version} - {date}
```

Comment:
Should we just leave "dev only" instead of {version} - {date}?
When there will be a new release, feature will be moved from dev to release list.
Or just put a note under the version-date that its the dev list

Reply:
Sure, I will add a note that this is the dev branch list and `{version} - {date}` should be updated when there is a new release. Thanks.

Reply:
Added `dev branch` instead of `{version} - {date}`